pom.xml

```diff
@@ -60,43 +60,12 @@
     <build>
         <finalName>${project.artifactId}</finalName>
         <plugins>
-            <plugin>
-                <groupId>org.wildfly.plugins</groupId>
-                <artifactId>wildfly-maven-plugin</artifactId>
-                <version>4.0.0.Beta4-SNAPSHOT</version>
-                <configuration>
-                    <feature-packs>
-                        <feature-pack>
-                            <location>org.wildfly:wildfly-galleon-pack:${version.wildfly}</location>
-                        </feature-pack>
-                        <feature-pack>
-                            <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.wildfly.cloud}</location>
-                        </feature-pack>
-                    </feature-packs>
-                    <layers>
-                        <layer>jaxrs-server</layer>
-                    </layers>
-                    <image>
-                        <push>true</push>
-                        <registry>quay.io</registry>
-                        <group>${user.name}</group>
-                    </image>
-                    <runtime-name>ROOT.war</runtime-name>
-                </configuration>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>image</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <configuration>
                     <systemProperties>
-                        <imageName>quay.io/${user.name}/${project.artifactId}</imageName>
+                        <imageName>quay.io/${user.name}/${project.artifactId}-${packaging}</imageName>
```

Comment:
does 'mvn clean verify` work with the wildfly-maven-plugin?
The image that is built is named `quay.io/foo/power-of-n` while this `imageName` would be resolved to `quay.io/foo/power-of-n-`.

One more argument to use the same name for the application image :)